jdtls.ext/com.microsoft.java.maven.plugin/src/main/java/com/microsoft/java/maven/PluginActivator.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
is it the right license?

Reply:
This repo is under MIT. Is there any required license for jdtls extension?

Reply:
I'll use EPL for consistency